include/radix/data/model/MeshLoader.hpp

```diff
@@ -1,24 +1,86 @@
 #ifndef RADIX_MESHLOADER_HPP
 #define RADIX_MESHLOADER_HPP
 
-#include <string>
-#include <map>
-
 #include <radix/data/model/Mesh.hpp>
 
+#include <map>
+#include <string>
```

Comment:
These includes here still aren't right, but the rest looks good now (just my reviews not the rest, the other reviewers get to decide on those).